dev/com.ibm.ws.microprofile.config.1.2_fat_tck/publish/tckRunner/tck/src/test/java/com/ibm/ws/microprofile/config12/test/ArchiveProcessor.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config12.test;
+
+import java.io.File;
+import org.jboss.arquillian.container.test.spi.client.deployment.ApplicationArchiveProcessor;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+
+/**
+ * We weave in the hamcrest jar that is used by some of the microprofile config tck tests.
+ * The build.gradle file pull the hamcrest jar from maven and puts it in the lib directory
+ */
+public class ArchiveProcessor implements ApplicationArchiveProcessor {
+
+    /* (non-Javadoc)
+     * @see org.jboss.arquillian.container.test.spi.client.deployment.ApplicationArchiveProcessor#process(org.jboss.shrinkwrap.api.Archive, org.jboss.arquillian.test.spi.TestClass)
+     */
+    @Override
+    public void process(Archive<?> applicationArchive, TestClass testClass) {
+        if (applicationArchive instanceof WebArchive) {
+            File hamcrest = new File("../../../lib/hamcrest-all-1.3.jar");
```

Comment:
Any additional jars used in the TCK that are not in LIberty can be added to build.gradle and referenced here.